src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,113 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static final int initialAge = 24;
+    private static final String initialLogin = "IvanIvan";
+    private static final String defaultLogin2 = "Misha Hrynko";
+    private static final String defaultLogin3 = "Bohdan Chupika";
+    private static final String initialPassword = "qwerty1234";
+    private static final String incorrectPassword = "12345";
+    private static final int incorrectAge = 15;
+    private static final int negativeAge = -5;
+    private static RegistrationService registrationService;
+    private User defaultUser = new User();

```

Comment:
let's create a new user in your setUp method directly

Reply:
I don't think that it's possible, because then other methods can't see defaultUser